src/containers/Trade.svelte

```diff
@@ -93,6 +94,17 @@
     ? TradeButtonStatus.EnterAmount
     : TradeButtonStatus.Trade;
 
+  // reset amounts and best order on network change
+  detectProvider('marina')
```

Comment:
we can use `marinaSrore.subscribe` and get network change from there?

Reply:
Yes, much better.

Implemented on e63cc1bf